### ISSUE TITLE
Handle merged documents

### ISSUE THREAD

**Issue by tsauerwein** [c2corg/v6_ui#238]
Once https://github.com/c2corg/v6_api/pull/189 is merged, the API will no longer return merged documents, that is documents where `redirects_to` is set (e.g. http://www.camptocamp.org/routes/640047/fr/depuis-les-baudes-boucle-par-le-plan-des-eaux-et-le-col-de-bounavalette).

When a merged document is requested, the API will return:

    {"redirects_to": 12345, "available_langs": ["en", "fr"]}

`available_langs` will be the locales of the document referenced in `redirects_to`.

Note that merged documents are not migrated any more, so the above example document will not be available.

**Comment by tsauerwein** [comment]
Duplicate of https://github.com/c2corg/v6_ui/issues/190
